cmd/preview/list.go

```diff
@@ -78,32 +91,29 @@ func List(ctx context.Context) *cobra.Command {
 	return cmd
 }
 
-func executeListPreviews(ctx context.Context, opts ListFlags) error {
-	oktetoClient, err := okteto.NewOktetoClient()
-	if err != nil {
-		return err
-	}
-	previewListOutput, err := getPreviewOutput(ctx, opts, oktetoClient)
-	if err != nil {
-		return err
+// executeListPreviews prints the list of previews
+func executeListPreviews(previews []previewOutput, outputFormat string) error {
+	if len(previews) == 0 {
```

Comment:
I think we need to log this only for plaintext format because if I use `-o json` or `yaml` this will cause a parsing error. For JSON/YAML returning empty array should be okay

Reply:
I've changed the approach, so now only the default view (table) is showing the message instead of the headers with empty table.

Yaml marshal returns `[]` for empty object, but for json i added a condition to show `[]` as the returned value of the marshal for empty object is null